components/BottomNavigation/src/private/MDCBottomNavigationItemView.m

```diff
@@ -139,7 +142,9 @@ - (void)centerLayoutAnimated:(bool)animated {
                     iconImageViewCenter.y - CGRectGetMidX(self.iconImageView.bounds));
     self.label.center =
         CGPointMake(CGRectGetMidX(self.bounds),
-                    CGRectGetMidY(self.bounds) + CGRectGetHeight(self.bounds) * 0.25f);
+                    CGRectGetMidY(self.bounds) + CGRectGetHeight(self.bounds) / 2 -
```

Comment:
Can this line be simplified to CGRectGetHeight(self.bounds)?

Reply:
Done.